.github/release-drafts/increasing-patch-version.yml

```diff
@@ -16,17 +13,12 @@ categories:
     labels:
       - 'dependencies'
 
-template: |
+header: |
   # :mega: Play SOAP $NEXT_PATCH_VERSION released!
 
   We are pleased to announce the release of SOAP module for [Play](https://playframework.org/) framework.
 
+template: |
   ## :green_book: What’s Changed
 
   $CHANGES
-
-  ## :bow: Credits
-
-  Thank you @scala-steward for the actual versions of dependencies :heart:
-
-  Special thanks to the following contributors who helped with this release: $CONTRIBUTORS
```

Comment:
@ihostage Instead of this credit section we will use the footer from the `.github` repo config

Reply:
Yep 👌